chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomTest.kt

```diff
@@ -0,0 +1,33 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import com.pusher.chatkit.state.JoinedRoom
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class JoinedRoomTest : Spek({
+
+    describe("given an empty joined room state") {
+        val initialState = ChatkitState(
+                joinedRoomsState = JoinedRoomsState(mapOf(), mapOf()))
```

Comment:
```suggestion
    describe("given no rooms") {
        val givenState = ChatkitState(joinedRoomsState = JoinedRoomsState(emptyMap(), emptyMap()))
```

Reply:
👍 Do you want me to go around the other tests and rename the `initiatlState` val as `givenState` too?

Reply:
yep, I think that would be better to not create confusion with the state created by `initial()` call